.github/workflows/intellijPluginUpload.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and Publish Intellij Plugin
+on: 
+  push: 
+jobs:
+  trigger_workflow:
+     strategy:
+       matrix:
+         os: [ubuntu-latest]
+     runs-on: ${{ matrix.os }}
+     env:
+         working-directory: ./tool-plugins/intellij
+     steps:
+       - uses: actions/setup-java@v1
+         with:
+           java-version: 1.8
+       - uses: actions/checkout@v2
+         with:
+           repository: ballerina-platform/ballerina-lang
+       - name: build and publish plugin
+         working-directory: ${{env.working-directory}}
+         run: |
+              ./gradlew buildPlugin
```

Comment:
@lasinicl ; Shall we include getting IntelliJ store credentials from Github secrets?